.github/CODEOWNERS

```diff
@@ -12,7 +12,7 @@
 #   can be optimized further.
 
 # Angular Migration team
-/core/templates/dev/head/services/contextual/logger.service.ts @bansalnitish
+/core/templates/dev/head/services/contextual/logger.service*.ts @bansalnitish
```

Comment:
If there aren't too many files, can you list them all?

Reply:
I see that the * pattern is followed by all the file. Is it better to follow this pattern or list the file and its spec file?

Reply:
If there aren't too many, then its probably better to list them all.

Reply:
Done.